### PR TITLE
fix: bonjour conditional loading

### PR DESCRIPTION
**Pull request by chmelevskij**
use `import()` for conditional loading of bonjour

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by rabbitambulance-ra** [comment]
This fixed my issue! (https://discord.com/channels/868013470023548938/930785604571107358/1187595587282010183)
Thanks, tomche!

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
